### ISSUE TITLE
when I run ctc_greedy_decoder function,how can I get the align information from all frames probability .

### ISSUE THREAD

**Issue by mengya22** [igormq/ctc_tensorflow_example#19]
As we kown,the ctc_greedy_decoder function can return a sequence that automatically eliminate the blank label and merge repeat labels. Now,I want get the message that contains complete greedy path information and probability before  eliminate the blank label and merge repeat labels.How can i get this information,thanks

**Comment by igormq** [comment]
Hi @mengya22 , what you want is the logits, located [here](https://github.com/igormq/ctc_tensorflow_example/blob/master/ctc_tensorflow_multidata_example.py#L120)
